hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -5324,22 +5328,25 @@ private long replayRecoveredEditsForPaths(long minSeqIdForTheRegion, FileSystem
   }
 
   private void handleException(FileSystem fs, Path edits, IOException e) throws IOException {
-    boolean skipErrors = conf.getBoolean(HConstants.HREGION_EDITS_REPLAY_SKIP_ERRORS,
-      conf.getBoolean("hbase.skip.errors", HConstants.DEFAULT_HREGION_EDITS_REPLAY_SKIP_ERRORS));
-    if (conf.get("hbase.skip.errors") != null) {
-      LOG.warn("The property 'hbase.skip.errors' has been deprecated. Please use "
-          + HConstants.HREGION_EDITS_REPLAY_SKIP_ERRORS + " instead.");
-    }
-    if (skipErrors) {
+    if (isSkipErrorEnabled()) {
       Path p = WALSplitUtil.moveAsideBadEditsFile(fs, edits);
-      LOG.error(HConstants.HREGION_EDITS_REPLAY_SKIP_ERRORS + "=true so continuing. Renamed "
-          + edits + " as " + p,
-        e);
+      LOG.info(
```

Comment:
LOG not right? Seems hbase.skip.errors=true may got here too.

Reply:
This is a behavior as expected?